build/tutorials/platform/getting-started-for-users.md

```diff
@@ -0,0 +1,121 @@
+
+Those are only a few in an ever-growing collection of dapps deployed on Avalanche. To find others, you can go to our official [ecosystem website](https://ecosystem.avax.network/marketplace). You can filter the projects by selecting the tags for the areas of your interest. There is also a [community-driven](https://www.avax-projects.com/) list of projects.
+
+Dip in, browse, try stuff. There are many gems in there.
+
+## Security
+
+As elsewhere in cryptocurrency space, you need to be keenly aware of the dangers. All transactions are final and irreversible, if you get scammed chances are nobody will be able to help you retrieve your funds.
+
+### Wallet Passphrase
+
+What is crucial to understand is that **your secret passphrase is your wallet**. Whoever has access to the secret 24 word passphrase has complete and full access and control over everything in the wallet. If you give someone your passphrase, you have given them everything in it. Therefore, never give your passphrase to anyone. Do not send it anywhere, do not type it into websites you found online or someone sent you the link to.
+
+The only place where you can enter the passphrase is into the official wallet, and even then, make sure you're on a secure network, that the website is genuine by typing the address wallet.avax.network address yourself, and checking the padlock to make sure that it really is wallet.avax.network. Any other site where you enter your passphrase into may easily steal all your funds. So be very, very careful about entering your passphrase anywhere. If in doubt, don't. Reach out to us to check if the site is legitimate.
+
+If you're working with non-trivial amounts of tokens (in other words, money you cannot comfortably lose), it is strongly advised to invest in a Ledger HW wallet. 
+
+### DYOR
+
+That stands for 'Do your own research'. In other words, don't just blindly trust anything you read online. Check for other sources, ask for second opinions. Be very careful and judicious with accepting news from one source.
+
+Be especially suspicious of people contacting you in private, offering help about issues you posted about publicly. Virtually every time it happens, that's a scammer trying to convince you to expose your passphrase, private keys or otherwise compromise your tokens.
+
+Also, don't rush into unknown projects that promise outsized returns. Any dapp you deposit your funds into has access to them. Google the project, see who is behind the project, check that it has traction in community. Check that the contracts are verified and audited. Look out for potential red flags.
+
+### Fake tokens
+
+Anyone can create a new token, and on Avalanche it is pretty cheap. Also, liquidity pool creation on DEXes is permissionless so anyone can create a new pair with a fake token that has the same name and token image as the real thing. To avoid that kind of scams, always select tokens from the official token lists on the DEXes, don't use links from other places.
+
```

Comment:
maybe in this section, mention that even though a user receives a token, it might not reflect in their wallet on the C-chain. That's because they need to add the contract address in order to see their balance.

a lot of people panic because they think they lost money when it doesn't show up in their wallet.

Reply:
added section with tokens.